autonomy/cli/mint.py

```diff
@@ -50,8 +55,8 @@
 )
 nft_decorator = click.option(
     "--nft",
-    type=str,
-    help="IPFS hash for the NFT image",
+    type=NFTArgument(),
+    help="IPFS hash or path for the NFT image",
```

Comment:
@DavidMinarsch This will allow the users to specify both NFT path and NFT hash when minting the protocol eg

```
autonomy mint ... --nft bayfbei...
```

Or 

```
autonomy mint ... --mint /path/to/nft.png
```